app/models/jira_issue.rb

```diff
@@ -17,8 +17,8 @@ class JiraIssue < ActiveRecord::Base
   validates :key, uniqueness: { message: 'Keys must be globally unique' }
   validates :key, format: { with: /\A.+-[0-9]+\z/ }
 
-  belongs_to :assignee, class_name: User, inverse_of: :commits, required: false
-  belongs_to :parent_issue, class_name: JiraIssue, inverse_of: :sub_tasks, required: false
+  belongs_to :assignee, class_name: User, inverse_of: :commits, required: false, null: true
+  belongs_to :parent_issue, class_name: JiraIssue, inverse_of: :sub_tasks, required: false, null: true
```

Comment:
I'm guessing many of these `belongs_to`s  can't be `null`?

Reply:
correct, and `null: false` is the default, so isn't needed to be added.  this is why the new migration is setting all the fields in the database to `null: false`

Reply:
Ok, so you found that just these 2 really need to be `null: true`?

Reply:
These at lease were the only ones that created test failures, and I believe the test coverage on this project is pretty solid.

Reply:
💯